### DISCUSSION TITLE
Improve email score on mail-tester.com

### DISCUSSION THREAD

**Discussion by BernhardBaumrock** [Leantime/leantime#1852] [General]
I've just installed leantime for the very first time and it looks great.

I've invited a new user to my project and got the email, great. Then I sent it to mail-tester.com and got a score of 8.2/10 which is not bad but it can be improved:

![image](https://github.com/Leantime/leantime/assets/8488586/2acc93d3-2eba-42b6-ab54-7eb1b4e88544)

Seems there are two main reasons for the penalty: 

-1.282 | HTML_IMAGE_ONLY_24 | HTML: images with 2000-2400 bytes of wordsYou should write more text in your email
-- | -- | --

-0.635 | HTML_MIME_NO_HTML_TAG | HTML-only message, but there is no HTML tagYour message should contain an <html> tag
-- | -- | --

**Comment by gloriafolaron** [comment]
Thank you for sending this! I wasn't aware of this testing platform. 

If you have any suggestions on specific items or have any room for contributing to the email designs, we would love the help there. This is one thing that we've been wanting to get to but it's been waiting for a larger overhaul of notifications in general.

**Comment by BernhardBaumrock** [comment]
Well I'm not an expert here, but wanted to share the result nonetheless. I guess it complains about the mail having only very little text and so the logo gets marked as "image only mail".

Not sure if that is really a problem. Not adding the logo would solve it technically, but it might not look as nice, so not sure what is the best way to do it.

**Reply by gloriafolaron** [reply]
The alternative may be to add some short content about powered by Leantime and about the tool -- to increase the length. 

For the open source user, that may not be ideal but can also be replaced with their own content. 

It may at least then serve as a prompt. 

I'll do some digging -- emails are really important for team adoption so it's worth improving. 

@marcelfolaron

**Reply by BernhardBaumrock** [reply]
Sounds good! Would be great if you can share your findings with us :)
